### DISCUSSION TITLE
Setting up an existing Rails project with Postgres

### DISCUSSION THREAD

**Discussion by 5280bm** [community/community#49533] [Codespaces]
### Select Topic Area

General

### Body

First, I love the concept of Codespaces. But, it seems darn near impossible to get an existing Rails app working with Postgres. When creating a codespace from an existing Rails app repository, I have updated the Ruby and added Postgres but continue to get role and permission errors. I have also tried to use devcontainer file - also with no success. Is there any clear documentation on how to get Rails working with Postgres?

**Comment by adnamalin** [comment]
Hi,

You can also try creating a rails app from our template: https://github.com/github/codespaces-rails. [Here](https://docs.github.com/en/codespaces/developing-in-codespaces/creating-a-codespace-from-a-template) is a doc for more information on templates.

**Reply by 5280bm** [reply]
Thank you Amanda! I definitely will go this route with brand new projects, but this particular one was an existing project that I wanted to move into Codespaces. Got everything going now.

**Reply by kwhandy** [reply]
Are you doing CRUD too or not? I'm facing difficulties for hours just for CRUD ops
